vampyr/functions/BoysFunction.cpp

```diff
@@ -0,0 +1,27 @@
+#include <array>
+
+#include "pybind11/eigen.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/functional.h"
+#include "pybind11/stl.h"
+
+#include "PyRepresentableFunction.h"
+
+#include "functions/BoysFunction.h"
+
+using namespace mrcpp;
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace vampyr {
+
+void boys_function(py::module &m, py::class_<RepresentableFunction<3>, PyRepresentableFunction<3>> &repfunc) {
```

Comment:
The Boys' function is 1D but it inherits from a 3D representable function. I think it should inherit from `PyRepresentableFunction<1>`. @stigrj @bjorgve ?